aggie-coding-club/Rev-Registration
# Created a How To Use page

## Description

This adds a new How To Use page that contains information and videos that help new users learn how to use the site. This page is accessed by clicking the "How To Use" link located near the "About Us" link in the landing page. 

Gifs need to be updated to match the current version of the site.

## Rationale

As new users may have trouble understanding how to use the site and take advantage of it's features, the new How To Use page should provide a way for users to quickly learn how to do so.

## How to test

I haven't made any automated tests for this as the content of the page is supposed to be static and there's not really any behavior to test. To make sure it's working you can just navigate to it.

## Screenshots

Only visible change to existing content is the addition of the link on the landing page. Here's how it looks. 

|Before|
![image](https://user-images.githubusercontent.com/16074716/119581298-e54a0480-bd87-11eb-9b0d-5fac64190552.png)
|After|![image](https://user-images.githubusercontent.com/16074716/119581158-acaa2b00-bd87-11eb-9859-46d92572f8dc.png)

## Related tasks

Closes #496
